src/main/java/io/vavr/jackson/datatype/deserialize/MaplikeDeserializer.java

```diff
@@ -63,15 +73,8 @@
                                                        TypeDeserializer elementTypeDeserializer,
                                                        JsonDeserializer<?> elementDeserializer);
 
-    @SuppressWarnings("unchecked")
     @Override
     public void resolve(DeserializationContext ctxt) throws JsonMappingException {
```

Comment:
now you can remove this method, because `MaplikeDeserializer` is abstract

Reply:
Actually I tried it this morning. The problem is that removing the method, I have to create another in the derived class `MapDeserializer`. Another solution is to remove `ResolvableDeserializer` interface from this base class, and only implement it in derived class `MultimapDeserializer`. WDYT?

Reply:
added a fix 03f0cc9